internal/compiler/tests/syntax/focus/initial_focus_non_input.slint

```diff
@@ -9,7 +9,8 @@ export X := Rectangle {
     callback trigger_focus_change();
     trigger_focus_change => {
         someRect.focus();
-//      ^error{focus\(\) can only be called on focusable elements}
+//      ^error{The expression is not a function}
+//               ^^error{Element 'Rectangle' does not have a property 'focus'}
```

Comment:
This looks slightly worse.
The fact that it is two error is another bug which is unrelated to this PR.

but `focus() can only be called on focusable elements` is better than `focus don't exist` kind of error.
This is not so important, but i wonder if we shouldn't still special-case that.

(but again, not a blocker for this PR)